### ISSUE TITLE
Add cluster_info field in resource abstractor

### ISSUE THREAD

**Issue by giobart** [oakestra/oakestra#326]
## Short
Can we add an optional `cluster_info` key:value list field in resource abstractor?
 
## Proposal
`cluster_info` can be a key:value list that each cluster can advertise. By default it's empty but for instance if we have Kubernetes cluster attached to Oakestra it can use this field to propagate some labels there. The info might be used by custom schedulers to make decisions that are not based on pre-defined values we're advertising. 

## Ratio
Makes cluster more general, giving them possibility of propagating information that are not part of the default implementation. 

## Impact
root resource abstractor.

## Development time
somewhere between 1 day and 99 years

## Status
Discussed with JakobKe

## Checklist
 
- [ ] Discussed
- [ ] Documented
- [ ] Implemented
- [ ] Tested

**Comment by giobart** [comment]
@melkodary what do you think?

**Comment by melkodary** [comment]
Like adding it to the schema? @giobart

**Comment by giobart** [comment]
Yes, exactly; simply adding it to the schema should do the trick, right?

**Comment by melkodary** [comment]
> Yes, exactly; simply adding it to the schema should do the trick, right?

Yes! Might I suggest also if we want it to be more generic to call it `resource_info` instead of `cluster_info` and move `cluster_name`  &  `cluster_location` under the object `resource_info`.
